docs/persistence/kafka/README.md

```diff
@@ -1,13 +1,24 @@
 # Kafka
 
-NAV uses Aiven hosted Kafka, the on-premise version of Kafka has been deprecated since 2021. Start using Kafka by creating a `Topic` resource in one of our Kubernetes clusters.
+NAV uses Aiven hosted Kafka, the on-premise version of Kafka has been deprecated since 2021 and decommisioned in october 2023.
```

Comment:
Maybe remove references to old on premises Kafka, as it's no longer with us. Also makes the doc read better for non-NAV tenants